jump-start-tutorial/exercise-5_route-to-contactslistcomponent.md

```diff
@@ -22,7 +22,7 @@ $ ng g c contacts-list
 1. Create a component `ContactsListComponent` with angular-cli and move the current contact list logic there from `ContactsAppComponent`
 2. Import `ContactsListComponent` in `src/app/app.module.ts` and add it to the module's `declarations`.
 2. Create `src/app/app.routes.ts`, import the `ContactsListComponent` and export an array with routes (e.g. `export const APP_ROUTES = [...]`), holding one route pointing to `ContactsListComponent`.
-3. In `app.module.ts` import `RouterModule` from `@angular/router` and `APP_ROUTES` from `./app/app.routes`
+3. In `app.module.ts` import `RouterModule` from `@angular/router` and `APP_ROUTES` from `./app.routes`
```

Comment:
If we really want to do that then it should be `./src/app/app.routes` . The `./app/` indicates that we're putting this file in there. `./app.routes` tells me it's the root of the projects (so next to `src`) which would be wrong..